common/fw-update-helper.cpp

```diff
@@ -383,19 +383,27 @@ namespace rs2
                 {
                     // stopping stream before starting fw update
                     auto fw_update_manager = dynamic_cast<firmware_update_manager*>(update_manager.get());
-                    std::for_each(fw_update_manager->get_device_model().subdevices.begin(), 
-                        fw_update_manager->get_device_model().subdevices.end(), 
+                    std::for_each(fw_update_manager->get_device_model().subdevices.begin(),
+                        fw_update_manager->get_device_model().subdevices.end(),
                         [&](const std::shared_ptr<subdevice_model>& sm)
                         {
-                            return sm->stop(fw_update_manager->get_viewer_model());
+                            if (sm->streaming)
+                            {
+                                try
+                                {
+                                    sm->stop(fw_update_manager->get_viewer_model());
+                                }
+                                catch (...) { }
```

Comment:
I assume this can throw (and has thrown?) which is why this is here?

If it throws, please add a debug/error message so we can see it in the logs...